src/libNode/FinalBlockProcessing.cpp

```diff
@@ -250,6 +235,10 @@ bool Node::VerifyFinalBlockCoSignature(const TxBlock& txblock)
         == false)
     {
         LOG_GENERAL(WARNING, "Cosig verification failed");
+        for (auto& kv : keys)
+        {
+            LOG_GENERAL(WARNING, "" << kv);
```

Comment:
same question

Reply:
This is to identify who cosign the message. This is important for debugging and such information should be captured when cosig verification fail.